### PR TITLE
search input added

### PR DESCRIPTION
**Pull request by aakash-sonova**
Added the search functionality to the input.

### CODE REVIEW

File: build/js/intlTelInput-jquery.js [hunk 1]
```diff
@@ -913,17 +931,25 @@
...
+                    } else if (searchCountry === true && e.keyCode >= 96 && e.keyCode <= 105) {
+                        _this9._searchCountry(e);
                     } else if (/^[a-zA-ZÀ-ÿа-яА-Я ]$/.test(e.key)) {
                         // jump to countries that start with the query string
-                        if (queryTimer) {
-                            clearTimeout(queryTimer);
+                        if (searchCountry) {
+                            _this9._searchCountry(e);
+                        } else {
+                            if (queryTimer) {
```
Comment (970dbarbato) [inline build/js/intlTelInput-jquery.js:943]: if query country is set, we never clear timeout?

File: src/css/intlTelInput.scss [hunk 2]
```diff
@@ -324,3 +324,21 @@ $mobilePopupMargin: 30px !default;
 .iti__flag.iti__np {
   background-color: transparent;
 }
+
+.iti__search_box {
+  position: fixed !important;
```
Comment (970dbarbato) [inline src/css/intlTelInput.scss:329]: Is there no proper way of setting without the !important?

**Comment by ghost** [comment]
I'm just wondering why there are file changes more than in their own branch `packiyarajpatchath:country_search_option`?
@aakash-sonova

**Comment by aakash-sonova** [comment]
> I'm just wondering why there are file changes more than in their own branch `packiyarajpatchath:country_search_option`? @aakash-sonova

Hi @977eibrahim , Its is so because they had made changes in the build folder however not in the pre-build folder - `src`. So I had to make the equivalent changes to `src`. Because otherwise once we build, the changes will be reverted. I believe `packiyarajpatchath` made changes in the wrong directory by mistake.

File: src/css/intlTelInput.scss (hunk 2 above)
Comment (aakash-sonova) [reply src/css/intlTelInput.scss:329]: As discussed yesterday, this style will be overridden from out component. @970dbarbato

**Review by 970dbarbato — APPROVED** [review]
LGTM

File: build/css/intlTelInput.css [hunk 3]
```diff
@@ -1275,4 +1275,22 @@
 
 .iti__flag.iti__np {
   background-color: transparent;
+}
+
+.iti__search_box {
+  position: fixed !important;
```
Comment (977mawais) [inline build/css/intlTelInput.css:1281]: Dont forget to add styles in the component

**Review by ghost — APPROVED** [review]

**Review by 977mawais — APPROVED** [review]
LGTM,

**Pull request merged**